src/validation.h

```diff
@@ -656,6 +654,12 @@ class CChainState
         return m_coins_views->m_dbview;
     }
 
+    //! @returns A reference to the mempool.
```

Comment:
This is wrong. It's returning a pointer, not a reference (and can't return a reference, since `m_mempool` may be null).